wls-gui-wahllokalsystem/src/components/common/inputs/BaseDateInput.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-text-field
+    :model-value="getFormattedValue(modelValue)"
+    :rules="[
+      REQUIRED,
+      (value) => DATE_NOT_BEFORE_WAHLTAG(value, currentUserWahltag),
+    ]"
+    label="Datum"
+    type="date"
+    clearable
+    data-test="base-date-input"
+    @update:model-value="onDateChanged"
+    @click:clear="resetDateKeepTime"
+  />
+</template>
+
+<script setup lang="ts">
+import type { PropType } from "vue";
+
+import { VTextField } from "vuetify/components";
+
+import { useDateTimeFormatter } from "@/composables/common/dateTimeFormatter.ts";
+import { useUserStore } from "@/stores/userStore.ts";
+import { DATE_NOT_BEFORE_WAHLTAG, REQUIRED } from "@/util/rules.ts";
+
+const { updateDateOfDateObject } = useDateTimeFormatter();
+const { currentUserWahltag } = useUserStore();
+
+const modelValue = defineModel({
+  type: Object as PropType<Date>,
+});
+
+const onDateChanged = (newDate: string) => {
+  const currentTime = modelValue?.value || new Date();
+  const updatedDate = updateDateOfDateObject(newDate, currentTime);
```

Comment:
sorry wenn der vorschlag so oft jetzt kam aber das verwirrt mich jedes mal wenn time da steht es aber um das date geht
```suggestion
  const currentDate = modelValue?.value || new Date();
  const updatedDate = updateDateOfDateObject(newDate, currentDate);
```